### PR TITLE
Adding the USWDS community page

### PR DESCRIPTION
**Pull request by jeremyzilar**
## Adding https://github.com/GSA/digitalgov.gov/issues/1125

This change adds in the USWDS listserv as a community of practice.

**Doc:** https://docs.google.com/document/d/1tLOz2HyxbqdAaFipA_wyTBp76fMkmooQcom4-Tw1MWY/edit

**Preview:** https://cg-06ab120d-836f-49a2-bc22-9dfb1585c3c6.app.cloud.gov/preview/gsa/digitalgov.gov/uswds-community/communities/uswds

### CODE REVIEW

File: content/communities/uswds.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+---
+
+{{< img-right src="uswds-logo" >}}
+
+The U.S. Web Design System (USWDS) is a library of code, tools, and guidance to help government teams design and build fast, accessible, mobile-friendly government websites backed by user research and modern best practices. [Read an overview »](https://designsystem.digital.gov/whats-new/updates/2019/04/08/introducing-uswds-2-0/)
+
+We’re proud that USWDS currently powers nearly [200 federal websites](https://designsystem.digital.gov/getting-started/showcase/all/) and we’re committed to making the design system work better for any federal website. With USWDS 2.0, we’re introducing a design system that’s built to grow with your needs — to help teams build better, more engaging websites for the American public.
+
+## Join
+Send an email to [uswds-subscribe-request@listserv.gsa.gov](mailto:uswds-subscribe-request@listserv.gsa.gov) with an empty subject line and empty email body.
```
Comment (thisisdano) [inline content/communities/uswds.md]: replace with: `Sign up to join the #uswds-public TTS Slack channel at  [chat.18f.gov](https://chat.18f.gov/)`

Comment (thisisdano) [reply content/communities/uswds.md]: For now, we're considering this the main community home

**Comment by jeremyzilar** [comment]
@Schultea Can you take a look at the community sign up language here?

File: content/communities/_index.md [hunk 2]
```diff
@@ -76,6 +76,9 @@ There are more than 10,000 people participating across government in 22 areas. O
 **[User Experience]({{< link "user-experience.md" >}})**: Developing successful government products and services through the practice of user-centered design, casting a broad tent in the user experience, customer experience, service design, innovation, behavioral design and other fields.<br/>
 [Learn&nbsp;more&nbsp;»]({{< link "user-experience.md" >}})
 
+**[U.S. Web Design System]({{< link "uswds.md" >}})**: An active, open-source community of government engineers, content specialists, and designers.<br/>
```
[the remark this answers is not in the document]
Comment (jeremyzilar) [reply content/communities/_index.md]: Done.

File: content/communities/uswds.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+
+**Community Managers:** Dan Williams, U.S. General Services Administration
+
+
+## Who We Are
+
+**USWDS** is an active, open source community of government engineers, content specialists, and designers. Our contributors, both in and out of government, support dozens of agencies and nearly 200 websites.
```
[the remark this answers is not in the document]
Comment (jeremyzilar) [reply content/communities/uswds.md]: Done.

File: content/communities/uswds.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+---
+url: /communities/uswds/
+uid: uswds
+title: 'U.S. Web Design System Community of Practice'
+summary: 'The federal government’s community for news and updates about the U.S. Web Design System'
```
[the remark this answers is not in the document]
Comment (jeremyzilar) [reply content/communities/uswds.md]: Done

File: content/communities/uswds.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  uid: uswds-2-illio-built-to-grow-alt
+  alt: ''
+
+---
+
+{{< img-right src="uswds-logo" >}}
+
+The U.S. Web Design System (USWDS) is a library of code, tools, and guidance to help government teams design and build fast, accessible, mobile-friendly government websites backed by user research and modern best practices. [Read an overview »](https://designsystem.digital.gov/whats-new/updates/2019/04/08/introducing-uswds-2-0/)
+
+We’re proud that USWDS currently powers nearly [200 federal websites](https://designsystem.digital.gov/getting-started/showcase/all/) and we’re committed to making the design system work better for any federal website. With USWDS 2.0, we’re introducing a design system that’s built to grow with your needs — to help teams build better, more engaging websites for the American public.
```
[the remark this answers is not in the document]
Comment (jeremyzilar) [reply content/communities/uswds.md]: Just checking that you want both graphs swapped out for the one you suggested, right?

File: content/communities/uswds.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+
+## What we do
+We support the USWDS’ efforts to make it easier to build accessible, mobile-friendly government websites for the American public. While our Listserv is primarily a means to broadcast news and updates, see the Related Resources section below for the many other ways we can work together to improve the system. 
+
+### Resources 
+
+- [The U.S. Web Design System](https://designsystem.digital.gov/) 
+- [USWDS on GitHub](https://github.com/uswds/uswds/) 
+- [Join the USWDS Slack](https://chat.18f.gov/)
```
[the remark this answers is not in the document]
Comment (jeremyzilar) [reply content/communities/uswds.md]: Talked with @afeijoo, and it makes sense to leave out the listserv for now to focus people on just joining the Slack

File: content/communities/uswds.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (jeremyzilar) [reply content/communities/uswds.md]: I swapped both out for the text that you suggested

File: content/communities/uswds.md (hunk 1 above)
Comment (jeremyzilar) [reply content/communities/uswds.md]: Done. I added in some additional copy about joining Slack, and what to expect.

**Comment by Schultea** [comment]
> @Schultea Can you take a look at the community sign up language here?

Are you adding listserv language or just keeping it to Slack?

**Comment by jeremyzilar** [comment]
@Schultea They are keeping it to just Slack for the time being :)

**Pull request merged**